clang/include/clang/Driver/Action.h

```diff
@@ -849,6 +869,30 @@ class StaticLibJobAction : public JobAction {
   }
 };
 
+/// Wrap all jobs performed between TFormInput (excluded) and Job (included)
+/// behind a `llvm-foreach` call.
+///
+/// Assumptions:
+///   - No change of toolchain, boundarch and offloading kind should occur
+///     within the sub-region;
+///   - No job should produce multiple outputs;
+///   - Results of action within the sub-region should not be used the wrapped
+///     region.
```

Comment:
```suggestion
///   - Results of action within the sub-region should not be used within the
///      wrapped region.
```
(assuming this was the intention)

Reply:
It is actually the opposite, the results of actions defined within the region are temp file lists, so cannot be used without using `llvm-foreach`.

I'll fix the comment.